### PR TITLE
[zh] Sync RBAC

### PR DESCRIPTION
**Pull request by Sea-n**
The file is already synced according to `./scripts/lsync.sh`

- Add aliases
- Re-sync some minor changes
- Add link anchors
- Fix punctuation
- Remove some redundant spaces

Netlify preview link: https://deploy-preview-33928--kubernetes-io-main-staging.netlify.app/zh/rbac
Production site: [zh](https://kubernetes.io/zh/docs/reference/access-authn-authz/rbac/), [en](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -1868,26 +2140,26 @@ In order from most secure to least secure, the approaches are:
      --namespace=my-namespace
    ```
 
-   <!--
+<!--
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:2143]: 这类标记还是按三个空格缩进为宜。

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 2]
```diff
@@ -2049,26 +2318,26 @@ RBAC 鉴权组件尝试对该 API 请求鉴权。如果 RBAC 也拒绝了该 API
...
-在服务器日志中没有出现 RBAC 拒绝消息，就可以删除 ABAC 鉴权器。
+一旦你[将角色授予服务账户](#service-account-permissions)且工作负载运行时，
+服务器日志中没有出现 RBAC 拒绝消息，就可以删除 ABAC 鉴权器。
 
 <!--
-### Permissive RBAC Permissions
+### Permissive RBAC permissions
 
-You can replicate a permissive policy using RBAC role bindings.
+You can replicate a permissive ABAC policy using RBAC role bindings.
```
Comment (ydFu) [inline content/zh/docs/reference/access-authn-authz/rbac.md:2340]: Please update the localization of this line.

Comment (Sea-n) [reply content/zh/docs/reference/access-authn-authz/rbac.md:2340]: How about this wording?
> 你可以使用 RBAC 角色绑定复制宽松的 ABAC 策略。

Comment (ydFu) [reply content/zh/docs/reference/access-authn-authz/rbac.md:2340]: LGTM,Thanks.
